AY2021S2-CS2103-T14-1/tp
# Docs(DG): Update storage diagram and add alias diagram/manual testing

## What this does
Closes #357 #349 #314 and #192
## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the DG has the changes
